src/pages/SensorEditPage.js

```diff
@@ -1 +1,295 @@
-export const SensorEditPage = () => null;
+/* eslint-disable react/jsx-wrap-multilines */
+import React from 'react';
+import { View, Text } from 'react-native';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import {
+  DataTablePageView,
+  EditorRow,
+  FlexRow,
+  HazardIcon,
+  InfoIcon,
+  PageButton,
+  Paper,
+  PaperSection,
+  SpacedChildren,
+  TextEditor,
+  BatteryIcon,
+  IconButton,
+  LightbulbIcon,
+  WifiIcon,
+} from '../widgets';
+import { BreachConfigRow } from './NewSensor/BreachConfigRow';
+import { DurationEditor } from '../widgets/StepperInputs';
+import { TextWithIcon } from '../widgets/Typography';
+
+import { useLoadingIndicator } from '../hooks/useLoadingIndicator';
+import { selectConfigs, selectSensorDetail } from '../selectors/sensorDetail';
+import { SensorDetailActions } from '../actions/SensorDetailActions';
+
+import { generalStrings, vaccineStrings } from '../localization';
+import {
+  MISTY_CHARCOAL,
+  APP_FONT_FAMILY,
+  DARKER_GREY,
+  LIGHT_GREY,
+  SUSSOL_ORANGE,
+  WHITE,
+} from '../globalStyles';
+
+import { SECONDS } from '../utilities/constants';
+import { VaccineActions } from '../actions/VaccineActions';
+
+const formatLastSyncDate = date => moment(date).fromNow();
+const formatBatteryLevel = batteryLevel => `${batteryLevel}%`;
+
+export const FridgeHeader = ({ macAddress, batteryLevel, lastSyncDate, onBlink }) => (
```

Comment:
I'm not sure what to do with this component. @Chris-Petty it is very similar to what you might be creating for the vaccine module page, but I am not sure if we should make a common header component (which I will also need for the fridge detail page) or if we should just do all 3 separate.

Just waiting to determine this before I refactor and remove the inline styles, potentially make it more generic/factor out to it's own file.